teachers_digital_platform/fixtures/tdp_initial_data.json

```diff
@@ -0,0 +1,562 @@
+        "fields": {
+            "parent": 2,
+            "title": "Paying taxes"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 4
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Choosing how to save"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 5
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Building emergency savings"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 6
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Saving for college"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 7
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Saving for long-term goals"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 8
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Saving for short-term goals"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 9
+    },
+    {
+        "fields": {
+            "parent": 3,
+            "title": "Investing"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 10
+    },
+    {
+        "fields": {
+            "parent": 4,
+            "title": "Managing risk"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 11
+    },
+    {
+        "fields": {
+            "parent": 4,
+            "title": "Preventing fraud and identity theft"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 12
+    },
+    {
+        "fields": {
+            "parent": 5,
+            "title": "Budgeting"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 13
+    },
+    {
+        "fields": {
+            "parent": 5,
+            "title": "Buying things"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 14
+    },
+    {
+        "fields": {
+            "parent": 5,
+            "title": "Paying bills"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 15
+    },
+    {
+        "fields": {
+            "parent": 5,
+            "title": "Paying for college"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 16
+    },
+    {
+        "fields": {
+            "parent": 6,
+            "title": "Getting loans"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 17
+    },
+    {
+        "fields": {
+            "parent": 6,
+            "title": "Managing credit"
+        },
+        "model": "teachers_digital_platform.activitysubtopic",
+        "pk": 18
+    },
+    {
+        "fields": {
```

Comment:
@wkrause1 ActivityGradeLevels: Sorry, I just realized the values for grade level is incorrect in the document I shared with you. The current values are:
1. High school (9-10)
2. High school (11-12)

Reply:
Gotcha, so instead of elementary, middle and high school we'd do the two you mentioned?